src/js/constants/ServiceSchema.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+let SERVICE_SCHEMA = {
```

Comment:
Well it is a schema in terms of what the schemaForm calls it. From this schema the formDefenition is created which translates to we could add the validation here.